fnol.d.ts

```diff
@@ -77,6 +79,8 @@ export type FNOLSubmittedPayload = {
   phone1Type:string 
   willEIGHandle:boolean
   isEmergency:boolean
+  chatbotUserFirstName?: string
```

Comment:
This will not be passed to the cabrillo APIs, just to the service. They are not modifying their APIs at all for this change